guidance_and_support/zendeskhelper.py

```diff
@@ -17,10 +17,19 @@ def generate_ticket(request, form, score=None, suspicious=False):
         spam_settings = SpamSettings.for_request(request)
         if score <= spam_settings.spam_threshold:
             return False
-    path = request.path
     email = form.cleaned_data.get('email')
     query = form.cleaned_data.get('query')
     name = form.cleaned_data.get('name', 'Anonymous requester')
+    path = request.path
+    if 'get-support' in path:
```

Comment:
In those cases I think it's not essential that we capture the previous page that someone was on, but it could be useful information anyway because it gives us two steps in the person's journey towards the point that they decided to contact us.

Reply:
Thanks @odscjames and @robredpath for your feedback.